src/com/epam/javatrainings/multiThread/Main.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.javatrainings.multiThread;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int minute = scanner.nextInt();
+        int second = scanner.nextInt();
+        if(Validation.isValid(minute,second)) {
+            MyTimer myTimer = new MyTimer(minute, second);
+            myTimer.run();
```

Comment:
The run method doesn't create a new thread, should be called start method

Reply:
I was just checking the work with debug, then I forgot to change the run function to the start function :(